batch/test/test_accounts.py

```diff
@@ -172,23 +172,6 @@ async def test_close_reopen_billing_project(dev_client: BatchClient, new_billing
     ], r
 
 
-async def test_close_billing_project_with_open_batch_errors(
```

Comment:
This raises an interesting question. What happens when a billing project is closed but a batch still has running jobs? A closed billing project should not receive any new bills. I think closing a billing project needs to cancel any "running" batches.

How should I think about the states of a batch after this change?